app/app/src/main/java/org/omecproject/up4/cli/FlowsStatsCommand.java

```diff
@@ -0,0 +1,108 @@
+            afterDownlink.put(dlUpfFlow.getSession().ueAddress(), dlUpfFlow.getCounter());
+        });
+
+        Map<Ip4Address, UpfCounter> afterUplink = Maps.newHashMap();
+        adminService.getUplinkFlows().forEach(ulUpfFlow -> {
+            afterUplink.put(ulUpfFlow.getTermination().ueSessionId(), ulUpfFlow.getCounter());
+        });
+
+        print(SEPARATOR);
+        if (!(beforeDownlink.keySet().equals(beforeUplink.keySet()) &&
+                beforeDownlink.keySet().equals(afterUplink) &&
+                beforeDownlink.keySet().equals(afterDownlink))) {
+            print("ERROR while reading stats!");
+            return;
+        }
+        for (Ip4Address ueSessionId : beforeDownlink.keySet()) {
+            if (!beforeUplink.containsKey(ueSessionId) ||
+                    !afterDownlink.containsKey(ueSessionId) ||
+                    !afterUplink.containsKey(ueSessionId)) {
+                print("UE session: " + ueSessionId.toString());
+                UpfCounter bfUl = beforeUplink.get(ueSessionId);
+                UpfCounter bfDl = beforeDownlink.get(ueSessionId);
+                UpfCounter afUl = afterUplink.get(ueSessionId);
+                UpfCounter afDl = afterDownlink.get(ueSessionId);
+                long droppedPktsUl = (afUl.getIngressPkts() - afUl.getEgressPkts()) -
+                        (bfUl.getIngressPkts() - bfUl.getEgressPkts());
+                long droppedPktsDl = (afDl.getIngressPkts() - afDl.getEgressPkts()) -
+                        (bfDl.getIngressPkts() - bfDl.getEgressPkts());
+
+                long txPktsUl = (afUl.getEgressPkts() - bfUl.getEgressPkts()) / SLEEP_TIME_MS;
+                long txBytesUl = (afUl.getEgressBytes() - bfUl.getEgressBytes()) / SLEEP_TIME_MS;
+                long txPktsDl = (afDl.getEgressPkts() - bfDl.getEgressPkts()) / SLEEP_TIME_MS;
+                long txBytesDl = (afDl.getEgressBytes() - bfDl.getEgressBytes()) / SLEEP_TIME_MS;
+
+                long rxPktsUl = (afUl.getIngressPkts() - bfUl.getIngressPkts()) / SLEEP_TIME_MS;
+                long rxBytesUl = (afUl.getIngressBytes() - bfUl.getIngressBytes()) / SLEEP_TIME_MS;
+                long rxPktsDl = (afDl.getIngressPkts() - bfDl.getIngressPkts()) / SLEEP_TIME_MS;
+                long rxBytesDl = (afDl.getIngressBytes() - bfDl.getIngressBytes()) / SLEEP_TIME_MS;
+
+                print("UPLINK");
+                print("    RX:  %s  %s", toReadable(rxBytesUl, "Bps"), toReadable(rxPktsUl, "pkt/s"));
+                print("    TX:  %s  %s", toReadable(txBytesUl, "Bps"), toReadable(txPktsUl, "pkt/s"));
+                print("    DROPPED:  %d", droppedPktsDl);
+                print("DOWNLINK");
+                print("    RX:  %s  %s", toReadable(rxBytesDl, "Bps"), toReadable(rxPktsDl, "pkt/s"));
+                print("    TX:  %s  %s", toReadable(txBytesDl, "Bps"), toReadable(txPktsDl, "pkt/s"));
+                print("    DROPPED:  %d", droppedPktsUl);
```

Comment:
Since OPs often asks about "throughput", Instead of showing 3 lines for RX, TX, DROPPED, would it easier to parse/more immediate if we showed a oneliner like: `tput: 15 Mbps / 250 pps (5% dropped)`?. Also note that this is bps, in Aether we usually use bits per second as the unit for bit rate, notByes per second.